docs/integration_testing.md

```diff
@@ -1,6 +1,6 @@
 # Integration Testing cAdvisor
 
-The cAdvisor integration tests can be found in `integration/tests`. These run queries on a running cAdvisor. To run these tests:
+The cAdvisor integration tests can be found in `integration/tests`. These queries run on a running cAdvisor. To run these tests:
```

Comment:
I don't think this is a typo. The original statement was essentially conveying the fact that the test runs queries against a running cadvisor instance.